src/Physics/InitialField.cpp

```diff
@@ -537,7 +537,7 @@ void seissol::physics::Ocean::evaluate(double time,
     if (std::abs(g - 9.81e-3) > 10e-15) {
       logError() << "Ocean scenario only supports g=9.81e-3 currently!";
     }
-    if (materialData.local.mu != 0.0) {
+    if (materialData.local.getMuBar() != 0.0) {
```

Comment:
Here, we used a greater than 1e-15 comparison above everywhere to compare to zero. I think we can do the same to stay consistent.